data.sql

```diff
@@ -58,4 +58,33 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smit
+
+/* Populating Vets table with data */
+
+INSERT INTO vets (name, age, data_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23'),('Maisy Smith', 26, '2019-01-17'),('Stephanie Mendez',64, '1981-05-04'),('Jack Harkness', 38, '2008-06-08');
+INSERT INTO specialization (species_id, vet_id) SELECT sp.id, ve.id FROM species sp, vets ve WHERE sp.name = 'Pokemon' AND ve.name = 'William Tatcher';
+INSERT INTO specialization (species_id, vet_id) SELECT sp.id, ve.id FROM species sp, vets ve WHERE (sp.name = 'Digimon' OR sp.name = 'Pokemon') AND ve.name = 'Stephanie Mendez';
+INSERT INTO specialization (species_id, vet_id) SELECT sp.id, ve.id FROM species sp, vets ve WHERE sp.name = 'Digimon' AND ve.name = 'Jack Harkness';
+
+/* Populating visits table with data */ 
+
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-05-24' FROM animals an, vets ve WHERE an.name = 'Agumon' AND ve.name = 'William Tatcher';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-07-22' FROM animals an, vets ve WHERE an.name = 'Agumon' And ve.name = 'Stephanie Mendez';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2021-02-02' FROM animals an, vets ve WHERE an.name = 'Gabumon' And ve.name = 'Jack Harkness';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-01-05' FROM animals an, vets ve WHERE an.name = 'Pikachu' And ve.name = 'Maisy Smith';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-03-08' FROM animals an, vets ve WHERE an.name = 'Pikachu' And ve.name = 'Maisy Smith';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-05-14' FROM animals an, vets ve WHERE an.name = 'Pikachu' And ve.name = 'Maisy Smith';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2021-05-04' FROM animals an, vets ve WHERE an.name = 'Devimon' And ve.name = 'Stephane Mendez';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2021-02-24' FROM animals an, vets ve WHERE an.name = 'Charmander' And ve.name = 'Jack Harkness';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2019-12-21' FROM animals an, vets ve WHERE an.name = 'Plantmon' And ve.name = 'Maisy Smith';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-08-10' FROM animals an, vets ve WHERE an.name = 'Plantmon' And ve.name = 'William Tatcher';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2021-04-07' FROM animals an, vets ve WHERE an.name = 'Plantmon' And ve.name = 'Maisy Smith';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2019-09-29' FROM animals an, vets ve WHERE an.name = 'Squirtle' And ve.name = 'Stephanie Mendez';
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT an.id, ve.id, '2020-10-03' FROM animals an, vets ve WHERE an.name = 'Angemon' And ve.name = 'Jack Harkness';
```

Comment:
- [ ] Good job so far 👏 👏 There is one piece of data that you forgot to record though 👀 which is the line below 👇 
```sql
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-04');
```
Angemon visited two times Jack Harkness You can check the data given again too. This one was not recorded. Please add this datum too 😃 😉